apps/styleguide/src/tables/ShowHide.vue

```diff
@@ -17,28 +17,30 @@
         )
       </b>
       <div v-for="(col, key) in columns" :key="key">
-        <b class="ml-0" v-if="col.name == 'mg_draft'">
```

Comment:
I only added a span around it, but thanks to indent it looks like a massive change